src/components/blog/blogs/BlogItem.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Typography, Paper, Box, Button } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles({
+  blogItemContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: '1rem',
+    height: '100%',
+    border: '1px solid #00000026',
+    borderRadius: '15px',
+    gap: '1rem',
+    '@media (max-width: 600px)': {},
+  },
+  blogContentWrapper: {
+    maxHeight: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    alignItems: 'start',
+    gap: '0.75rem',
+    '@media (max-width: 600px)': {},
+  },
+  blogItemTextWrapper: {
+    flex: 1,
+    maxHeight: '100%',
+    padding: '0.5rem',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    alignItems: 'start',
+    gap: '0.75rem',
+    '@media (max-width: 600px)': {},
+  },
+  blogCategory: {
+    fontSize: '12px',
+    letterSpacing: '1px',
+    color: '#ADADAD',
+    lineHeight: 'normal',
+    fontWeight: 600,
+    '@media (max-width: 600px)': {},
+  },
+  blogTitle: {
+    // fontSize: '18px',
+    lineHeight: 'normal',
+    fontWeight: 600,
+    '@media (max-width: 600px)': {},
+  },
+  blogContent: {
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    display: '-webkit-box',
+    '-webkit-line-clamp': 4,
+    '-webkit-box-orient': 'vertical',
+    '@media (max-width: 600px)': {},
+  },
+  blogItemBtn: {
+    padding: 0,
+    color: '#000',
+    fontSize: '12px',
+    textTransform: 'none',
+    '@media (max-width: 600px)': {},
+  },
+});
+
+interface BlogItemProps {
```

Comment:
@UzeeChamp1 pls start the interface name with capital i(I)
```suggestion
interface IBlogItemProps {
```